clients/lightrider/lightrider-entry.js

```diff
@@ -61,6 +61,11 @@ export async function runLighthouseInLR(connection, url, flags, lrOpts) {
       config.settings.onlyCategories = categoryIDs;
     }
   }
+  // Set to 0 to disable
```

Comment:
Set what to 0? Who is this comment for?

Reply:
Oh, I think I see. Could you instead make `megaBidx` be optional, or union with `| false` ?

Reply:
moved it to `channel='fr' && queryParam('bidx')` instead.